portfolio/src/main/java/com/google/sps/data/StateDessertVotes.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+
+/** Dessert vote totals for a state. */
+public final class StateDessertVotes {
+
+  private final String state;
+  private final int cheesecakeVotes;
+  private final int applePieVotes;
+  private final int chocolateChipCookiesVotes;
+  private final int tiramisuVotes;
+  private final int chocolateCakeVotes;
+
+  public StateDessertVotes(String state, int cheesecakeVotes, int applePieVotes, 
```

Comment:
Maybe you can extract a Dessert Enum class to wrap these choices. If you choose to do so, you can refactor it in a following PR.